### PR TITLE
Add 'Create Cosmos DB Account' command

### PR DESCRIPTION
**Pull request by EricJizbaMSFT**
This only supports the kind 'Mongo' and uses the same '+' icon as the 'Add Server' command. Both of these will have to be fixed in the future.

Related #11, #24

### CODE REVIEW

File: src/commands.ts [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+        }
+    }
+}
+
+export class LocationQuickPick implements vscode.QuickPickItem {
+    readonly label: string;
+    readonly description: string;
+    constructor(readonly location: SubscriptionModels.Location) {
+        this.label = location.displayName;
+        this.description = location.name;
```
Comment (whiskeyjay) [inline src/commands.ts]: Is it necessary to show name on UI?

Comment (EricJizbaMSFT) [reply src/commands.ts]: Nope

File: src/commands.ts [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+        });
+
+        if (resourceGroupName) {
+            const locationPick = await this.getLocation(subscriptionPick, "Select a location to create your Resource Group in...");
+
+            if (locationPick) {
+                return vscode.window.withProgress({ location: vscode.ProgressLocation.Window }, async (progress) => {
+                    progress.report({ message: `Cosmos DB: Creating resource group '${resourceGroupName}'` });
+                    const resourceManagementClient = new ResourceManagementClient(subscriptionPick.session.credentials, subscriptionPick.subscription.subscriptionId);
+                    return resourceManagementClient.resourceGroups.createOrUpdate(resourceGroupName, { location: locationPick.location.name });
```
[the remark this answers is not in the document]
Comment (EricJizbaMSFT) [reply src/commands.ts]: If the group already exists in the same location, then it uses that group. If the group already exists in a different location, then an error message pops up saying something like "An existing resource group already exists in a different location"

If you don't like that behavior, then I'll file an issue and address it in a later commit

[the remark this answers is not in the document]
Comment (whiskeyjay) [reply src/commands.ts]: Since a list of existing resource groups is already there, might as well check against it.

**Pull request merged**